src/Commands/core/LocaleCommands.php

```diff
@@ -208,6 +209,78 @@ public function exportValidate(CommandData $commandData): void
         }
     }
 
+    /**
+     * Imports multiple translations from the defined directory.
+     *
+     * @throws \Exception
+     */
+    #[CLI\Command(name: self::IMPORT_ALL, aliases: ['locale-import-all', 'locale:import:all'])]
+    #[CLI\Argument(name: 'directory', description: 'The path to directory with .po files to import.')]
+    #[CLI\Option(name: 'type', description: 'String types to include, defaults to <info>not-customized</info>. Recognized values: <info>not-customized</info>, <info>customized</info>')]
+    #[CLI\Option(name: 'override', description: 'Whether and how imported strings will override existing translations. Defaults to the Import behavior configured in the admin interface. Recognized values: <info>none</info>, <info>customized</info>, <info>not-customized</info>, <info>all</info>')]
+    #[CLI\Usage(name: 'drush locale:import-all /var/www/translations', description: 'Import all custom translations from the defined directory.')]
+    #[CLI\Usage(name: 'drush locale:import-all --types=customized --override=all', description: 'Import all custom translations from the defined directory and override any existing translation.')]
+    #[CLI\ValidateModulesEnabled(modules: ['locale'])]
+    public function importAll($directory, $options = ['type' => self::OPT, 'override' => self::OPT])
+    {
+        if (!is_dir($directory)) {
+            throw new \Exception('The destination directory does not exist.');
+        }
+
+        // Look for .po files in defined directory
+        $poFiles = glob($directory . DIRECTORY_SEPARATOR . '*.po');
+        if (empty($poFiles)) {
+            throw new \Exception('Translation files not found in the destination directory.');
+        }
+
+        $this->getModuleHandler()->loadInclude('locale', 'translation.inc');
+        $this->getModuleHandler()->loadInclude('locale', 'bulk.inc');
+
+        $translationOptions = _locale_translation_default_update_options();
+        $translationOptions['customized'] = $this->convertCustomizedType($options['type']);
+        $override = $this->convertOverrideOption($options['override']);
+        if ($override) {
+            $translationOptions['overwrite_options'] = $override;
+        }
+        $langcodes_to_import = [];
+        $files = [];
+        foreach ($poFiles as $file) {
+            // Ensure we have the file intended for upload.
+            if (file_exists($file)) {
+                $poFile = (object) [
+                    'filename' => basename($file),
+                    'uri' => $file,
+                ];
+                $poFile = locale_translate_file_attach_properties($poFile, $translationOptions);
+                if ($poFile->langcode == LanguageInterface::LANGCODE_NOT_SPECIFIED) {
+                    $this->logger->warning(dt('Can not autodetect language of file @file', [
```

Comment:
Add an explanation of the required file name pattern. `some-name.langcode.po`

Reply:
Added details about pattern